packages/scss/src/components/readMore/vars.scss

```diff
@@ -0,0 +1,6 @@
+@mixin vars {
+	--components-readMore-lineClamp: 5;
+	--components-readMore-link-display: flex;
+	--components-readMore-link-backgroudColor: var(--pr-t-elevation-surface-default);
```

Comment:
Discussed with Clément: We should have _raised_ color by default as it's going to be the main usage.

Reply:
✔️